meetings/2022-10-26.md

```diff
@@ -0,0 +1,151 @@
+* SC: General problem is structured error messages.
+
+
+#### Structured Error Messages
+URL: https://github.com/solid/specification/issues/28
+* SC: Rough agreement that error messages should be in RDF and client can find people you can to request access for, or something else. Specific consideration is: can we do that in RDF? Do we need a data model to have messages be useful for clients? Example 403 can be many different reasons. Can we be more explicit on what the error is? Some security considerations... how much to reveal about the error. Key thing is having a model to provide that. 
+* SC: If needed to, familiarize yourself with the issues/PRs. 
+
+
+### Use IRIs for resource identification
+URL: https://github.com/solid/specification/issues/347
+* SC: Currently Solid protocol expresses things using URIs, partly because of some of the requirements are starting off HTTP and not necessarily from an RDF perspective. This issue is about bringing focus to IRI and whether they would be mapped to URIs and round tripping.
+* SC: Example: you need to give access to someone's WebID. Request comes in, should the server give rights to the IRI version or the URI version in the authorization rule, and would the server be doing the mapping between the two. In RDF it'd be one or the other. Would server take request as is, or do the mapping. If one or the other, they might not match unless there is a mapping. We need wider insight from a bigger circle re: security considerations. 
+* WT: Elaborate on example. Why such a transition be necessary if client asks ??? request containing an IRI why should there be a translation, if it is just an identifier at that point?
+* SC: When you make an HTTP request, what's going over the wire is just a URI. If it was originally an IRI the client will map it to a URI, so the server is getting a URI. Question is for the server, do I convert to IRI, or take as is? If as is, and put it in an access control resource, how would that be interoperable? Solid protocol doesn't specify whether that should occur. One server might do the mapping, another server might not. There would be a mismatch. 
+* WT: That scenario would be an unlimited set of scenarios, because most identifiers (IRIs or not) are sufficient. Only when an IRI needs to pass in HTTP that it has to be translated. 
+* HS: I've found the inbox discovery from earlier more real. Sounds like you need to gather use cases with problems that would arise. URIs are meant to be just IRIs encoded in a particular way. Doesn't make a different how that is encoded locally.
+* SC: If one server find a match they might grant access. Same rule used on a different server, they might not match. 
+* HS: Server should be consistent. 
+* WT: Authorization example: if my ACL I have IRI of some agent, a request to that resource bound by that ACL would contain an IRI, not a URI because it is not part of the HTTP protocol, it is in the body. 
```

Comment:
```suggestion
* WT: Authorization example: if in my ACL I have the IRI of some agent, a request to the resource bound by that ACL would contain an IRI, not a URI, because it is not part of the HTTP protocol, it is in the body or token. 
```